jquery.dynatable.js

```diff
@@ -114,6 +114,23 @@
       sortTypes: {},
       records: null
     },
+    ajax: {
+      ajaxComplete: function(response) {
```

Comment:
I'd just call these settings `success` and `complete` instead of `ajaxComplete` and `ajaxDone` just to keep naming consistent and reduce redundancy (since they're located in the `ajax` object, setting `ajax.success` and `ajax.complete` is a bit cleaner than `ajax.ajaxComplete`, etc.